recipe/meta.yaml

```diff
@@ -25,14 +25,14 @@ requirements:
 
```

Comment:
Is it still worth keeping unix selector for gnuconfig, or we can remove it too?

Reply:
I've removed it